health/health-checks/src/main/java/io/helidon/health/checks/HeapMemoryHealthCheck.java

```diff
@@ -144,5 +151,19 @@ public Builder thresholdPercent(double threshold) {
             this.threshold = threshold;
             return this;
         }
+
+        /**
+         * Set up the heap space health check via config key, if present.
```

Comment:
The javadoc should contain a table of config keys used, description and default value. 
Example:
```
Configuration options:
             * <table class="config">
             * <caption>Secret configuration</caption>
             * <tr>
             *     <th>key</th>
             *     <th>default value</th>
             *     <th>description</th>
             *     <th>builder method</th>
             * </tr>
             * <tr>
             *     <td>{@value CONFIG_KEY_THRESHOLD_PERCENT_SUFFIX}</td>
             *     <td>{@code 99.9}</td>
             *     <td>Percentage of heap memory that needs to be full for this health check to fail</td>
             *     <td>{@link #tresholdPercent(double}</td>
             * </tr>
             * </table>
```

Reply:
Added for both classes.